ci-operator/step-registry/cucushift/logging/cucushift-logging-commands.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function show_test_execution_time() {
```

Comment:
function `show_test_execution_time` and `echo_e2e_tags` are not used, can be removed to simplify the code.